docs/source/visualisation/visualise_charts_with_plotly.md

```diff
@@ -1,60 +1,53 @@
 # Visualise charts in Kedro-Viz
 
-## Visualisation with Plotly
-Kedro-Viz supports [Plotly](https://plotly.com/python/) and [Matplotlib](https://matplotlib.org/) integrations so that you can share your data insights while exploring your pipeline.
-
-[Plotly](https://plotly.com/python/) is a free and open source Python library that allows you to make interactive, publication-quality graphs. With the Plotly integration on Kedro-Viz, you can output your interactive charts as part of your pipeline visualisation.
-
-We have also used the Plotly integration to allow users to [visualise metrics from experiments](../logging/experiment_tracking.md).
+This page describes how to make interactive visualisations of your Kedro projects with Kedro-Viz, which supports integration with [Plotly](https://plotly.com/python/) and [Matplotlib](https://matplotlib.org/).
 
+## Visualisation with Plotly
 
-You must update the `requirements.txt` file in your Kedro project and add the following datasets to enable Plotly for your project.
+We use the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) and add a reporting pipeline that uses Plotly. Even if you have not yet worked through the tutorial, you can still follow this example; you'll need to use the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) to generate a copy of the project with working code in place:
 
-`kedro-datasets[plotly.PlotlyDataSet, plotly.JSONDataSet]~=1.0.0`
+From your terminal:
 
+```bash
+kedro new --starter=spaceflights
+```
 
-You can view Plotly charts in Kedro-Viz when you use Kedro's plotly datasets.
+When prompted for a project name, you can enter any name, but we will assume `Kedro Tutorial` throughout.
 
-There are two types of Plotly datasets in Kedro: the `plotly.PlotlyDataSet` and `plotly.JSONDataSet`.
+### Update the dependencies
 
-### `plotly.PlotlyDataSet`
+There are two types of Plotly datasets supported by Kedro:
 
-To use this dataset, configure your plot in the `catalog.yml` file. This dataset only supports [Plotly Express](https://plotly.com/python/plotly-express).
+* `plotly.PlotlyDataSet` which only supports [Plotly Express](https://plotly.com/python/plotly-express)
+* `plotly.JSONDataSet` which supports Plotly Express and [Plotly Graph Objects](https://plotly.com/python/graph-objects/)
 
-Below is an example of how to visualise plots on Kedro-Viz using `plotly.PlotlyDataSet`.
+To use the Plotly datasets, you must update the `requirements.txt` file in the `src` folder of your Kedro project to add the following dependencies:
 
-The below functions can be added to the `nodes.py` and `pipeline.py` files respectively.
+```{note}
+At the time of writing, the tutorial works up to Kedro version 0.18.3 so you need to pin the versions as shown.
```

Comment:
It's a good point, but it's hard to know how many have installed Kedro at this point and how many are still to do so. I would avoid too much complexity because the pin will ensure they have the correct version (at the risk of a re-install just for those that come fresh to Kedro at this point).